nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -155,6 +155,8 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - [Monado](https://monado.freedesktop.org/), an open source XR runtime. Available as [services.monado](#opt-services.monado.enable).
 
+- [intel-gpu-tools](https://drm.pages.freedesktop.org/igt-gpu-tools), tools for development and testing of the Intel DRM driver. Available as [hardware.intel-gpu-tools](#opt-hardware.intel-gpu-tools.enable)
```

Comment:
igt isn't really a service? but I'm not sure about how release notes are handled for non-service modules, just keeping this as a note.

Reply:
Yeah, I was a bit unsure about this too, but historically setcap wrappers like this have also been added to the new services section. It seems like this section has always been catch all for any new module whether or not it's actually a service.

Reply:
20.09 is the only release that further breaks it down into subsections: https://github.com/NixOS/nixpkgs/blob/a0c8b880a41471e9754b31af84e8ea52132943c2/nixos/doc/manual/release-notes/rl-2009.section.md#new-services-sec-release-2009-new-services